assets/js/modules/adsense/components/settings/AdBlockingRecoveryApp/steps/PlaceTagsStep.js

```diff
@@ -98,20 +109,24 @@ export default function PlaceTagsStep( { setActiveStep } ) {
 			return;
 		}
 
+		await trackEvent( `${ viewContext }_adsense-abr`, 'setup_enable_tag' );
+
 		setActiveStep( ENUM_AD_BLOCKING_RECOVERY_SETUP_STEP.CREATE_MESSAGE );
 	}, [
 		saveSettings,
 		setActiveStep,
 		setAdBlockingRecoverySetupStatus,
 		setUseAdBlockingRecoverySnippet,
 		syncAdBlockingRecoveryTags,
+		viewContext,
 	] );
 
-	// Error snippet option is enabled by default.
 	useMount( () => {
+		// Error snippet option is enabled by default.
 		if ( ! useAdBlockingRecoveryErrorSnippet ) {
 			setUseAdBlockingRecoveryErrorSnippet( true );
 		}
+		trackEvent( `${ viewContext }_adsense-abr`, 'setup_place_tag' );
```

Comment:
I don't think we want to track the default state of the control as an event – we may need to check with @marrrmarrr about this. AFAIK we don't do it for others.

Reply:
This is not the default state of the control, it's the event for user viewing step one of ABR setup. The control event is `setup_enable_tag`. Although `setup_place_tag` could be better named, ie. `view_place_tag_step`.